end2end-tests/ktor/build.gradle.kts

```diff
@@ -62,10 +66,24 @@ tasks {
         listOf("--http-controller-opts", "AUTHENTICATION")
     )
 
+    val generateKtorInstantDateTimeCode = createCodeGenerationTask(
+        "generateKtorInstantDateTimeCode",
+        "src/test/resources/examples/instantDateTime/api.yaml",
+        listOf("--type-overrides", "DATETIME_AS_INSTANT", "--serialization-library", "KOTLINX_SERIALIZATION")
+    )
+
+    val generateKtorQueryParametersCode = createCodeGenerationTask(
+        "generateKtorQueryParametersCode",
+        "src/test/resources/examples/queryParameters/api.yaml",
+        listOf("--type-overrides", "DATETIME_AS_INSTANT", "--serialization-library", "KOTLINX_SERIALIZATION")
```

Comment:
The type override has not effect when Kotlinx Serialization is used and can be removed: It is always `kotlinx.datetime.Instant` ([source](https://github.com/cjbooms/fabrikt/blob/master/src/main/kotlin/com/cjbooms/fabrikt/model/KotlinTypeInfo.kt#L92)).